packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -85,6 +85,8 @@ import { DraggablePanel } from '@veupathdb/coreui/dist/components/containers';
 import { TabbedDisplayProps } from '@veupathdb/coreui/dist/components/grids/TabbedDisplay';
 import { GeoConfig } from '../../core/types/geoConfig';
 import Banner from '@veupathdb/coreui/dist/components/banners/Banner';
+import DonutMarkerComponent from '@veupathdb/components/lib/map/DonutMarker';
+import ChartMarkerComponent from '@veupathdb/components/lib/map/ChartMarker';
```

Comment:
Interested in the motivation for the rename...

Oh, I see, because of the DonutMarker SVG Icon component.

This could be a general "style guide" discussion at our next meeting maybe.